lib/mirage_block_partition.ml

```diff
@@ -45,19 +47,20 @@ module Make(B : Mirage_block.S) = struct
     let sector_start = Int64.add sector_start b.sector_start in
     let sector_end = Int64.add sector_start num_sectors in
     sector_start >= b.sector_start && sector_end <= b.sector_end
+      let connected = true (* or false, depending on your needs *)
```

Comment:
This is a constant different from the field defined on line 9. This is always going to be true. This binding is what is referred to in `read` and `write`.